src/server.js

```diff
@@ -100,6 +113,14 @@ in ${config.public ? "public" : "private"} mode`);
 
 		manager.autoloadUsers();
 	}
+
+	function shutdown() {
+		log.info("Quitting...");
+		server.close(); // Removes socket file
+		process.exit();
+	}
+
+	process.on("SIGINT", shutdown);
```

Comment:
I'm wondering if server `close` event is fired when node shuts down. https://nodejs.org/api/http.html#http_event_close